membership/Maintainers.md

```diff
@@ -0,0 +1,11 @@
+# GeoPandas Project Maintainers
+
+The list of Maintainers is managed at the subproject level. Not all subprojects
+have created a list of Maintainers yet. The list of subproject Maintainers will
+be listed here once available.
+
+[GeoPandas repository maintainers](https://geopandas.org/en/stable/about/team.html)
+
+[Contextily contributors](https://contextily.readthedocs.io/en/latest/#contributors)
+
+[xyzservices contributors](https://xyzservices.readthedocs.io/en/stable/#contributors)
```

Comment:
```suggestion
[GeoPandas repository maintainers](https://geopandas.org/en/latest/about/team.html)

[Contextily contributors](https://contextily.readthedocs.io/en/latest/#contributors)

[xyzservices contributors](https://xyzservices.readthedocs.io/en/latest/#contributors)
```

Contextily leads to `latest` while the other to the `stable` version of the docs. Given the governance is not related to releases, I would point them all to the latest.